src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,29 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login called with parameters: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.warn("Authentication failed for user {}", login);
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User {} successfully logged in", login);
         return user;
     }
 
     private User findByLogin(String login) {
+        logger.debug("Searching for user with login {}", login);
```

Comment:
The debug log in the `findByLogin` method helps trace the user search process, which is beneficial for debugging.